### ISSUE TITLE
Relay return values to another neuron

### ISSUE THREAD

**Issue by samuei** [kalliope-project/kalliope_neuron_rss_reader#2]
Long story short, I'm hoping to take the `items` list returned from this neuron and pass them into a command using the shell neuron. However, I am having a devil of a time figuring out how to get the information from one place to the other. 

More specifically, I have an RSS feed with an `enclosure` element (which feedparser puts into an `enclosures` list on the item) that links to some audio. I'm hoping to take that element's url property and pass it out to mplayer to play the audio. It's possible there's an easier way to get at this than what I've described above.

I would appreciate any thoughts or assistance.

**Comment by royto** [comment]
Hi @samuei , 

It may be possible (to verify) by combining [kalliope_memory](https://kalliope-project.github.io/kalliope/brain/brain/#kalliope_memory) and [neurotransmitter](https://kalliope-project.github.io/kalliope/brain/neurons/neurotransmitter/)

It could be something like that: (to adapt to your case ;) )

```yaml
- name: "play-audio-url"
    signals:
      - order: "play url {url]"
    neurons:
      - mplayer:
          url: "http://myaudiofile.com"

  - name: "play-enclosure-from-feed"
    signals:
      - order: "play enclosure from feed"
    neurons:
      - rss_reader:
          feed_url: "http://myfeed.com"
          kalliope_memory:
            enclosure_url: "feed["path"]["to"]["enclosure"]["url"]"
      - neurotransmitter:
          direct_link: "play-audio-url"
          url: "{{ kalliope_memory['enclosure_url'] }}"
```

You should go to the gitter chat of the project to have help about that => https://gitter.im/kalliope-project/Lobby

**Comment by samuei** [comment]
That was very helpful! Thank you.

Unfortunately, I found myself unable to send the `items[0].enclosures[0].url` object into kalliope_memory. Sending `items` worked well enough, but by the time it reached the "play-audio-url" synapse, it had been converted from a List to a string. Since I would have to run a script to isolate the url I'm looking for in the string anyway, I've decided to just short-circuit the whole thing and run a script in the first place.
